frontend/apps/reader/skimtowidget.lua

```diff
@@ -248,7 +248,7 @@ function SkimToWidget:init()
             else
                 page = self.ui.bookmark:getNextBookmarkedPageFromPage(self.curr_page)
             end
-            self:gotoBookmarkCallback(page)
+            self:gotoBookmark(page)
```

Comment:
Plus the case :angel: 
```suggestion
            self:goToBookmark(page)
```